### ISSUE TITLE
Multiple panels home layout should be saved across restart

### ISSUE THREAD

**Issue by johnfocker** [OpenLauncherTeam/openlauncher#166]
#### General information

* **Device:** Galaxy S7
* **Android Version:** 7.0
* **App version:** v0.5.1

#### Steps to reproduce

1. Add several panels with icons and widgets on each one
2. Restart OpenLauncher
3. Icons/widgets layout is messed up: icons and widgets from one panel will overlap icons or widgets from one another

#### Expected result
Layout is saved across restart or phone reboot.

**Comment by dkanada** [comment]
Could you possibly show some screenshots from before and after the reboot? This should be working but maybe we missed an edge case or something.

**Comment by MFlisar** [comment]
Have achieved that too. By just having one desktop with icons and then I moved a new icon to desktop and dragged it left to a new page. The moving left to a new page seemed to be important but I've not further investigated it. That's all I remember.

**Comment by rainerzufall** [comment]
Happens to me too. very reproducible :(

Here is what I do (similar to johnfocker):
- Add 5 panels. selected the middle one as "Home".
- make backup.
- restart OpenLauncher. (via menu or go into other launcher und then open OL again)
- everything is scrambeled, especially the widgets.
- restoring backup does not help. 

Here I have made some screenshots:

initial setup:
![before restart](https://user-images.githubusercontent.com/12666598/30317622-ea1501f6-97aa-11e7-959a-29b894fe86b8.PNG)

after restart:
![after restart](https://user-images.githubusercontent.com/12666598/30317785-75a25570-97ab-11e7-98df-bf17fc22b814.PNG)
("Home" is now the second panel from left [with the note "left side 1st"]; Widgets from the right and the clock are in the middle panel.)

after backup:
![after restore of backup](https://user-images.githubusercontent.com/12666598/30317798-7e0949a8-97ab-11e7-84a4-76607cf1c47b.PNG)
("Home still is second from left"; clock just moves to "Home" and the right position; but since "Home" is wrong, and the other settings are also not restored, that does not help either )

I hope this helped in finding the culprit. I did not find a debug log option in the app, so if I can provide any other information to solve this, I would be happy to!

ps. thanks for all the work on this launcher! It is a really great one, but unfortunately until this issue is resolved I can not use it :/
